src/service/notification-api/shared/FamilyHubs.Notification.Api.Contracts/MessageDto.cs

```diff
@@ -1,11 +1,15 @@
-﻿namespace FamilyHubs.Notification.Api.Contracts;
+﻿// ReSharper disable PropertyCanBeMadeInitOnly.Global
+// ReSharper disable NonReadonlyMemberInGetHashCode
+// ReSharper disable UnusedMember.Global
+// ReSharper disable UsageOfDefaultStructEquality
```

Comment:
Are these ReSharper comments just a temporary thing that will be addressed later?

Reply:
There are nearly 2000 warnings overall. Some are simple like namespaces and others are more complicated. In this case its attached to an object that gets serialized so I don't want to change everything to get, init and in one property, its written to.